packages/client/ui/Chip/Chip.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import {Close} from '@mui/icons-material'
+import {twMerge} from 'tailwind-merge'
+
+interface Props {
+  label: string
+  picture?: string | null
+  icon?: React.ReactElement | null
+  className?: string
+  onDelete?: (event: any) => void
```

Comment:
+1 looks like the type is: `onDelete?: (event: React.MouseEvent<SVGSVGElement, MouseEvent>) => void`